radixdlt/build.gradle

```diff
@@ -63,10 +63,12 @@ jacocoTestReport {
 dependencies {
     implementation 'io.reactivex.rxjava3:rxjava:3.0.0'
     implementation 'com.sleepycat:je:18.3.12'
-    implementation 'com.github.radixdlt:radix-engine-library:rc~1.0-beta.5-SNAPSHOT'
+//    implementation 'com.github.radixdlt:radix-engine-library:rc~1.0-beta.5-SNAPSHOT'
```

Comment:
I assume this is for debugging purposes, should be changed back

Reply:
Yes, as I've said three times :D please ignore any gradle stuff, will be changed last, @clstrfsck will help me with that 👍